### ISSUE TITLE
usage issue

### ISSUE THREAD

**Issue by guicamarotto** [snapappointments/bootstrap-select#2494]
Hello, I am trying to use bootstrap-select in a .net core project with razor but can't use.

I have this libs installed and added direct on my project

```
{
      "library": "font-awesome@5.12.0",
      "destination": "wwwroot/lib/font-awesome/"
    },
    {
      "library": "jquery@3.5.0",
      "destination": "wwwroot/lib/jquery/"
    },
    {
      "library": "twitter-bootstrap@4.4.1",
      "destination": "wwwroot/lib/bootstrap/"
    }
```

this are my references on master layout page

```
<link href="~/lib/bootstrap/css/bootstrap.css" rel="stylesheet" />
    <link href="~/lib/font-awesome/css/fontawesome.css" rel="stylesheet" />
    <link href="~/lib/font-awesome/css/brands.css" rel="stylesheet" />
    <link href="~/lib/font-awesome/css/solid.css" rel="stylesheet" />
    <link rel="stylesheet" href="~/css/site.css" />
<script src="~/lib/jquery/jquery.js"></script>
    <script src="~/lib/bootstrap/js/bootstrap.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/notify/0.4.2/notify.min.js"></script>
    <script src="~/js/site.js" asp-append-version="true"></script>
```

and this are my references added to a specific page:

```
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.20/css/dataTables.bootstrap4.min.css" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.4.1/css/bootstrap-datepicker3.css" />

    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/css/bootstrap-select.min.css">

<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.4.1/js/bootstrap-datepicker.min.js"></script>

    <script type="text/javascript" src="https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/1.10.20/js/dataTables.bootstrap4.min.js"></script>

    <!-- Latest compiled and minified JavaScript -->
    <script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/js/bootstrap-select.min.js"></script>
```

but when I try to convert my select object, I get this error

```
<select id="vendorSelect" class="selectpicker" title="Choose one of the following..." data-live-search="true" >
   <option value=""></option>
</select>
```

![image](https://user-images.githubusercontent.com/10661352/86280051-7aad3a80-bbb1-11ea-9615-90c160a1cbc1.png)

**Comment by caseyjhol** [comment]
This is an issue with jQuery 3.5.0, and it was fixed in bootstrap-select v1.13.15. See https://github.com/snapappointments/bootstrap-select/releases.

**Comment by guicamarotto** [comment]
nice! thanks!
